install/example/freebsd_freenas.md

```diff
@@ -88,13 +89,7 @@ Open the server.xml configuration file:
 ```
 # nano /usr/local/apache-tomcat-8.5/conf/server.xml
 ```
-Find the following element:
-```
-<Host name="localhost" appBase="webapps"
-    unpackWARs="true" autoDeploy="true"> 
-```
-Add the line *<Context docBase="airsonic" path="" reloadable="true" />* below the element, before the
-</Host> tag; then save:
+Goto line 154 and add *<Context docBase="airsonic" path="" reloadable="true" />* below the element; then save.
```

Comment:
My gut feeling is that it's much more future-proof to provide instructions for finding the line (possibly optionally with the line number) in case this changes. This will help for the next update!